apps/condo/domains/billing/schema/ValidateQRCodeService.js

```diff
@@ -65,43 +87,50 @@ const ValidateQRCodeService = new GQLCustomSchema('ValidateQRCodeService', {
         },
         {
             access: true,
-            type: 'type ValidateQRCodeOutput { qrCodeFields: JSON! }',
+            type: 'type ValidateQRCodeLastReceiptDataOutput { id: ID!, period: String!, toPay: String! }',
+        },
+        {
+            access: true,
+            type: 'type ValidateQRCodeFeesOutput { explicitServiceCharge: String, explicitFee: String }',
+        },
+        {
+            access: true,
+            type: 'type ValidateQRCodeOutput { qrCodeFields: JSON!, lastReceiptData: ValidateQRCodeLastReceiptDataOutput, explicitFees: ValidateQRCodeFeesOutput!, amount: String! }',
```

Comment:
Is there any option to serialize qrCodeFields?

Reply:
Maybe, but this field is decoded fron input string. Input string content may be different for different qr-codes.